crypto/src/keymanager.rs

```diff
@@ -0,0 +1,28 @@
+use blsttc::{PublicKeySet, SecretKeySet};
+
+use crate::keypair::KeyPair;
+
+// KeyManager is a single trusted dealer that generates keys for a group of users.
+// It is responsible for holding public key set and secret key set.
+pub struct KeyManager {
```

Comment:
I think we don't need keymanager unless we will distribute new keypairs whenever new node comes in. It would be sufficient to put pk and sk per nodes in the config file.

Reply:
You're right. Distributing keypairs will only happen at very first moment of setting up a network, so this explicit keymanager should be fixed. Thanks!